crates/hdk/src/lib.rs

```diff
@@ -165,6 +165,10 @@
 //!   - Close runs when an agent is deprecating an old source chain in favour of a new one.
 //!   - All zomes in a DNA migrate at the same time.
 //!   - Any failure fails the migration.
+//! - `fn recv_remote_signal(signal: SerializedBytes) -> ExternResult<()>`:
+//!   - Allows the guest to receive remote signals sent from other agents via the [`send_remote_signal`](crate::p2p::send_remote_signal) host function.
+//!   - Only receives signals from a coordinator zome of the same name in the remote agent's cell.
+//!   - The single input parameter of this callback can be an arbitrary type with a `serde::Deserialize + std::fmt::Debug` implementation, rather than `SerializedBytes`. If you choose to do this, the call will fail if the type cannot be deserialized from the serialized bytes.
```

Comment:
Doesn't it need the `SerializedBytes` implementation also? That's what I would guess at least but I haven't tested it.